spec/models/info_request_spec.rb

```diff
@@ -1058,6 +1058,29 @@
       info_request.expire(:preserve_database_cache => true)
     end
 
+    it 'updates the search index' do
+      expect(info_request).to receive(:reindex_request_events)
+      info_request.expire
+    end
+
+    it 'removes itself from the varnish cache' do
+      expect(info_request).to receive(:purge_in_cache)
+      info_request.expire
+    end
+
+    it 'removes any zip files' do
+      allow(info_request).to receive(:foi_fragment_cache_directories) {[]}
```

Comment:
Maybe worth a comment here to say that if we don't do this, we'll get spurious calls to remove other directories?